rero_ils/modules/patron_transactions/jsonschemas/patron_transactions/patron_transaction-v0.0.1.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Patron Transaction",
+  "description": "JSON schema for Patron Transaction.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate Patron Transaction records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/patron_transactions/patron_transaction-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Patron Transaction ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "creation_date": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Patron Transaction creation date"
+    },
+    "note": {
+      "title": "Patron Transaction Note",
+      "description": "Additional informations about patron transaction",
+      "type": "string",
+      "minLength": 3
+    },
+    "status": {
+      "title": "Status",
+      "type": "string",
+      "enum": [
+        "open",
+        "closed"
+      ],
+      "default": "open",
+      "form": {
+        "options": [
+          {
+            "label": "open",
+            "value": "open"
+          },
+          {
+            "label": "closed",
+            "value": "closed"
+          }
+        ]
+      }
+    },
+    "type": {
+      "title": "Patron Transaction Type",
+      "type": "string",
+      "enum": [
+        "overdue",
+        "photocopy",
+        "subscription"
+      ],
```

Comment:
maybe add a type `other` to allow more possibilities

Reply:
+1 about "other" in transaction Type 
Please also add an entry for "Interlibrary loan". It think it could help later....